src/vs/workbench/workbench.main.ts

```diff
@@ -442,6 +442,7 @@ import 'sql/workbench/parts/dataExplorer/browser/dataExplorerViewlet';
 import 'sql/workbench/parts/dataExplorer/browser/dataExplorerExtensionPoint';
 import 'sql/workbench/parts/dataExplorer/electron-browser/nodeActions.contribution';
 
+import 'sql/platform/telemetry/telemetry.contribution';
```

Comment:
Add a // {{SQL CARBON EDIT}} tag on top of this line since it's VSCode's base

Reply:
All of these lines are ours - the line with the SQL CARBON EDIT is on 433 above.

Reply:
Looks good to me then. I'll be adding more telemetry events in the contribution file in future.